### DISCUSSION TITLE
Add last hit date column to com_redirect links list view

### DISCUSSION THREAD

**Discussion by motwguy** [joomla/joomla-cms#22753] [2. Cold Features]
### Steps to reproduce the issue
After a site change a website may receive a number hits from cached Google results that will go away after Google's cache is updated. There is a 404 Hits column in the report, but adding a Last Hit Date column would help identify when a redirect was no longer needed.

**Comment by richard67** [comment]
You are talking about the "Redirects" component, com_redirect, and its redirect links list, "administrator/index.php?option=com_redirect", right?

**Comment by motwguy** [comment]
Yes.

**Comment by richard67** [comment]
@motwguy I've updated the title of your feature request so it's more clear what you mean.

**Comment by motwguy** [comment]
thanks!
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22753">issues.joomla.org/tracker/joomla-cms/22753</a>.</sub>
